staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/v1beta1/BUILD

```diff
@@ -0,0 +1,33 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
```

Comment:
what is generated and what is not generated in this package.  How I regenerate the generated bits?

Reply:
Do you mean add scripts similar with "hack/update-generated-runtime.sh" and "hack/update-generated-runtime-dockerized.sh" to generate the "pb.go" file?

Reply:
> Do you mean add scripts similar with "hack/update-generated-runtime.sh" and "hack/update-generated-runtime-dockerized.sh" to generate the "pb.go" file?

Yes.  Or however you created it.  This package looks generated and we need to know how to generate it if we want to update it.

Reply:
@wu-qiang did you address this?

Reply:
yes, "hack/update-generated-kms.sh" and "hack/update-generated-kms-dockerized.sh" are added for it.